scripts/locale/scenario_49_allies.fr.po

```diff
@@ -0,0 +1,3547 @@
+"The Arlenians are your allies. The Kraylors and Exuari are your enemies and "
+"will try to prevent you from completing your missions. Duration: "
+"approximately 30 minutes\n"
+"\n"
+"Version 1 - Nov2021\n"
+"\n"
+"Three missions available in current version. Tested with EE version 20190910"
+msgstr ""
+"- Durée approx. : 30mn par mission\n"
+"- Difficulté : <<…>> (variable)\n"
+"- Vaisseaux joueurs : 1 à plusieurs\n"
+"\n"
+"<<TRADUCTION EN COURS : 11%>>\n"
+"\n"
+"Les Arléniens sont vos alliés.\n"
+"Les Kraylors et Éxuaris sont vos ennemis et tenteront de vous empêcher de "
+"terminer vos missions.\n"
+"3 missions sont disponibles dans la version actuelle.\n"
+"\n"
+"Version 1 - nov 2021\n"
+"\n"
+"Titre original : Allies and Enemies\n"
+"Auteur : Xansta"
+
+# Scenario setting
+msgctxt "setting"
+msgid "Enemies"
+msgstr "Ennemis"
+
+msgctxt "setting"
+msgid "Configures the amount of enemies spawned in the scenario."
+msgstr "Définit le nombre d'ennemis générés dans le scénario."
+
+msgctxt "Enemies"
+msgid "Easy"
+msgstr "Facile"
+
+msgctxt "Enemies"
+msgid "Easy goals and/or enemies"
+msgstr "Objectifs et/ou quantité d'ennemis faciles."
+
+msgctxt "Enemies"
+msgid "Normal"
+msgstr "Normal"
+
+msgctxt "Enemies"
+msgid "Normal amount of enemies. Recommended for a normal crew."
+msgstr "Quantité d'ennemis par défaut. Recommandé pour un équipage standard."
+
+msgctxt "Enemies"
+msgid "Hard"
+msgstr "Difficile"
+
+msgctxt "Enemies"
+msgid "Hard goals and/or enemies"
+msgstr "Objectifs et/ou quantité d'ennemis difficiles."
+
+msgctxt "Enemies"
+msgid "One"
+msgstr "Mission unique"
+
+msgctxt "Enemies"
+msgid ""
+"Only one mission randomly chosen from several possible missions (Default is "
+"all missions in random order)"
+msgstr ""
+"Une seule mission choisie au hasard parmi 3 (par défaut c'est les 3 missions "
+"dans un ordre aléatoire)."
+
+msgctxt "Enemies"
+msgid "Easy One"
+msgstr "Mission unique facile"
+
+msgctxt "Enemies"
+msgid ""
+"Easy goals and/or enemies, only one mission randomly chosen from several "
+"possible missions (Default is all missions in random order)"
+msgstr ""
+"Objectifs et/ou quantité d'ennemis faciles. Une seule mission choisie au "
+"hasard parmi 3 (par défaut c'est les 3 missions dans un ordre aléatoire)."
+
+msgctxt "Enemies"
+msgid "Hard One"
+msgstr "Mission unique difficile"
+
+msgctxt "Enemies"
+msgid ""
+"Hard goals and/or enemies, only one mission randomly chosen from several "
+"possible missions (Default is all missions in random order)"
+msgstr ""
+"Objectifs et/ou quantité d'ennemis difficiles. Une seule mission choisie au "
+"hasard parmi 3 (par défaut c'est les 3 missions dans un ordre aléatoire)."
+
+#: scripts/scenario_49_allies.lua:64
+msgctxt "buttonGM"
+msgid "Mining"
+msgstr ""
+
+#: scripts/scenario_49_allies.lua:66
+msgctxt "buttonGM"
+msgid "Admiral"
+msgstr ""
+
+#: scripts/scenario_49_allies.lua:68
+msgctxt "buttonGM"
+msgid "Doomsday"
+msgstr ""
+
+#: scripts/scenario_49_allies.lua:408
```

Comment:
I'm totally naïve about PO files, and poedit, etc, and was just wondering if/how these comments are auto-generated and whether anything depends on the line number mentioned in the comment? 

ie. If I were to edit the allies scenario so that this comms happened on another line, is there any need to update all the .po files as well? I'm assuming not, but wonder why they are mentioned in the first place

Reply:
@daid and @Xansta will be able to answer you. I just take care of translating into French. ^^

Reply:
Yeah, they are autogenerated, I think by the update_locale build target.
You assumed correctly. If the line number changes but not the translatable contend, the translation would still work.
However, they are still quite useful for translating. I use them every now and then, when I am unsure about the exact context of a translatable string. So I can look at the code (the poedit editor has a convenient function for this) which usually helps me to understand in what circumstance that text is used, so it gives me clues how I could translate it.

Reply:
Thanks! So the line numbers could diverge, but would be fixed up during the build. Perfect, thanks!